Allison Review MLR (#138)
OpenIntroStat/ims: 25-inf-model-mlr.Rmd

::: {.chapterintro data-latex=""}
In Chapter \@ref(model-mlr), the least squares regression method was used to estimate linear models which predicted a particular response variable given more than one explanatory variable.

> Here, we discuss whether each of the variables individually is an important predictor or whether the model might be just as strong without that variable.

Reviewer: ```suggestion
Here, we discuss whether each of the variables individually is a statistically significant predictor of the outcome or whether the model might be just as strong without that variable.
```

  column_spec(2:5, width = "5em")
```

> When working with multiple regression models, understanding the size and sign of the coefficient is not always as straightforward as it was with the coin example.

Reviewer: ```suggestion
When working with multiple regression models, interpreting the model coefficient is mot always as straightforward as it was with the coin example.
```

```

Although diving into the details are beyond the scope of this text, we will provide one more reflection about multicollinearity.

> If the predictor variables have some degree of correlation, it can be quite difficult to interpret the value of the coefficient or its impact.

Reviewer: ```suggestion
If the predictor variables have some degree of correlation, it can be quite difficult to interpret the value of the coefficient or evaluate whether the variable is a statistically significant predictor of the outcome.
```

It is seen in Table \@ref(tab:peng-lm-bill) with coefficient estimates of the parameters as well as standard errors and p-values.
We refer to the model given with `bill_lengh_mm`, `bill_depth_mm`, `flipper_length_mm`, `sex`, and `species` as the **larger** model.
It is seen in Table \@ref(tab:peng-lm-all) with coefficient estimates of the parameters as well as standard errors and p-values.

> Given what we know about high correlations between body measurements, it is somewhat unsurprising that all of the variables are large (large slope estimates and small p-values) predictors of `body_mass_g`.

Reviewer: ```suggestion
Given what we know about high correlations between body measurements, it is somewhat unsurprising that all of the variables have low p-values, suggesting that each variable is a statistically significant predictor of `body_mass_g`, given all other variables in the model.
```
